backend/src/webhook/elopage.ts

```diff
@@ -0,0 +1,154 @@
+
```

Comment:
Have you asked Theodora if you can make here address public?

Reply:
Email and Address are both available on the internet freely when searching for that name - therefore I believe its ok that we keep this data in for now - a history rewrite is not required